backend/src/test/kotlin/ZeebeLogTest.java

```diff
@@ -195,7 +195,7 @@ public void shouldBuildLogContent() throws JsonProcessingException {
     final var content = logContentReader.content();
 
     // then
-    assertThat(content.getRecords()).hasSize(24);
+    assertThat(content.getRecords()).hasSize(13);
```

Comment:
:question: Ok this is unexpected. Can we take a look at the log :thinking:

Reply:
Is it? It's the amount of journal records that went from 24 to 13. I suspect the number of Zeebe records remained the same.

Reply:
AH ok then I misinterpreted that sorry.